### ISSUE TITLE
suggestion allow for filenames in lowercase (.jpeg, .pdf, .png)

### ISSUE THREAD

**Issue by coliff** [sapegin/textlint-rule-terminology#56]
The following are flagged as issues:

```
kitten.jpg
report.pdf
puppy.png
```

because JPG, PDF and PNG must be in uppercase. I'd like to suggest that filetypes shouldn't be flagged as an issue if they follow on from a `.` (like HTML is right now). I don't know the exact RegEx for this - would it be the same as the HTML one?

```
["(?<!\\.)jpeg\\b", "JPG"],
["(?<!\\.)png\\b", "PDF"],
["(?<!\\.)png\\b", "PNG"],
```

Let me know, if you agree and the above is the correct RegEx I'd be happy to open a quick PR. :-) 
(I think .CSS and .GIF could be added to the list too).
